salt/modules/opkg.py

```diff
@@ -129,7 +129,7 @@ def version(*names, **kwargs):
     return __salt__['pkg_resource.version'](*names, **kwargs)
 
 
-def refresh_db():
+def refresh_db(failhard=False):
```

Comment:
This change was based off of `aptpkg.py`:

https://github.com/saltstack/salt/blob/develop/salt/modules/aptpkg.py
```
def refresh_db(cache_valid_time=0, failhard=False):
```
So not all the modules are consistent.